account+view pages updated and account page added (#241)
OctoMind-dev/mintlify-docs: account-view/projects.mdx

---
title: Projects
description: "All about projects"
icon: "plus"
---

A project is a collection of test cases and test reports tied to a website or web app. The **test target URL** you define for it is used as a basis for generating test steps. You can extend it to subpaths of your website during test case creation.

You can run your test **against a different URL** while keeping the subpaths - the URL is "templated". This allows you to [run your test cases against multiple deployments](/execute-test-cases#1-trigger-a-test-run) of the same application.

<Frame caption="test target url of a project, 08/2024">
  <img
    src="images/accounts/test-target.png"
    alt="test target url of a project"
  />
</Frame>

> Projects are listed in the left menu bar underneath your organization. The active project is shown in the top menu to the right

Author: ```suggestion
Projects are listed in the hamburger menu under your organization.
```
